docs/frontend_api.md

```diff
@@ -305,6 +308,20 @@ A utility method that can be used to safely parse the text content (handles null
 Ferdium.safeParseInt(mySelector.innerText)
 ```
 
+### isImage(url)
+
+A utility method that can be used to verify if a url is an image. Returns `1` if is image and `0` if it is not an image.
```

Comment:
Rather than a `1` or `0` - can it be made to return `true` or `false` (boolean data type)?